TombEngine/Renderer/Renderer11.h

```diff
@@ -90,11 +91,26 @@ namespace TEN::Renderer
 	{
 		int ObjectNumber;
 		int RoomNumber;
-		Vector3 Position;
+		int IndexInRoom;
+		Pose Pose;
 		Matrix World;
 		Vector4 Color;
 		Vector4 AmbientLight;
 		std::vector<RendererLight*> LightsToDraw;
+		std::vector<RendererLight*> CachedRoomsLights;
```

Comment:
Please rename this vector to `CachedRoomLights`. In English, you don't need to make previous word plural, if last one is also plural.

Reply:
I am a donkey in english sometimes :-D agreed